templates/security/disa-stig.html

```diff
@@ -41,14 +41,14 @@ <h2>What is DISA-STIG?</h2>
 
   <div class="u-fixed-width">
     <h2>The DISA-STIG guides for Ubuntu</h2>
-    <p class="u-sv3">DISA has, in conjunction with Canonical, developed STIGs for Ubuntu. The U.S. DoD provides the <a class="p-link--external" href="https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux">STIG checklist</a>, which can be viewed using <a class="p-link--external" href="https://public.cyber.mil/stigs/stig-viewing-tools/">STIG viewer</a>, and <a class="p-link--external" href="https://public.cyber.mil/stigs/scap/">SCAP content for auditing</a>. The versions of Ubuntu that have STIGs available by DISA are marked on the table below.</p>
+    <p class="u-sv3">Together with Canonical DISA has, developed STIGs for Ubuntu. The U.S. DoD provides the <a class="p-link--external" href="https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux">STIG checklist</a>, which can be viewed using <a class="p-link--external" href="https://public.cyber.mil/stigs/stig-viewing-tools/">STIG viewer</a>, and <a class="p-link--external" href="https://public.cyber.mil/stigs/scap/">SCAP content for auditing</a>. The versions of Ubuntu that have STIGs available by DISA are marked on the table below.</p>
```

Comment:
```suggestion
    <p class="u-sv3">Together with Canonical DISA has developed STIGs for Ubuntu. The U.S. DoD provides the <a class="p-link--external" href="https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux">STIG checklist</a>, which can be viewed using <a class="p-link--external" href="https://public.cyber.mil/stigs/stig-viewing-tools/">STIG viewer</a>, and <a class="p-link--external" href="https://public.cyber.mil/stigs/scap/">SCAP content for auditing</a>. The versions of Ubuntu that have STIGs available by DISA are marked on the table below.</p>
```

Reply:
mmm I think your suggestion is removing the comma. The PR has the comma in it, it's in green.

Reply:
Oh wait, I see, it should be moved after Canonical